app/src/main/java/com/strv/movies/MainActivity.kt

```diff
@@ -46,13 +44,94 @@ class MainActivity : ComponentActivity() {
                                 DarkLightModeSwitchIcon(isDarkTheme = isDarkTheme)
                             }
                         )
+                        Greeting("MOVIES")
```

Comment:
It is good practice to put all static texts to `strings.xml` resource file and then get them in Compose by `stringResource(id = R.string.app_name)` (replace app_name by name of string you defined in `strings.xml`). Because then you can create separate xml file for each language and Android will automatically switch you app to language chosen in system settings.